### ISSUE TITLE
Qwen image generation and quality output issue using the Qwen lighting 8 steps Lora

### ISSUE THREAD

**Issue by Shangooriginal** [comfyanonymous/ComfyUI#9297]
### Custom Node Testing

- [x] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

I believe the recent frontend template update to Comfyui has created a problem or bug that is affecting the quality output of images generated with Qwen image model GGUF Q4 and Qwen lighting 8 steps lora.

Yesterday, evening I was able to generate images with the  correct quality output. But after updating comfyui today, the image quality is bad, distorted and weird using the same prompt, settings and seed. What I did notice when I was looking at the code terminal is that the Qwen lighting 8 steps lora (i.e. lora key not loaded) is NOT been loaded correctly by Comfyui.

I don't know if the lora key not loaded issue happened yesterday too since I didn't look at the code terminal since I was getting the quality image output  I wanted.  

<img width="1152" height="896" alt="Image" src="https://github.com/user-attachments/assets/40094012-f1c1-44c9-b81a-164fb1ed9c54" />

### Actual Behavior

See above comment.

### Steps to Reproduce

See above comment.

### Debug Logs

```powershell
lora key not loaded
```

**Comment by machineminded** [comment]
Experiencing similar issues.  In my case the lightning lora works but other loras do not load.  Is this an issue with how it was trained?

**Comment by Alright-Man** [comment]
Same issue here, quite possibly the lora keys not being loaded as I'm getting the same. 
lora key not loaded: transformer_blocks.0.attn.add_k_proj.alpha
lora key not loaded: transformer_blocks.0.attn.add_k_proj.lora_down.weight
lora key not loaded: transformer_blocks.0.attn.add_k_proj.lora_up.weight
lora key not loaded: transformer_blocks.0.attn.add_q_proj.alpha

**Comment by Glzchat-liang** [comment]
8.12更新的问题，使用之前的版本就恢复了

**Comment by machineminded** [comment]
I ran a script to inspect the keys from a couple LoRA.  It looks like the Qwen-provided 8- and 4-step loras all have keys that look like this...

```
transformer_blocks.9.txt_mlp.net.0.proj.lora_down.weight
transformer_blocks.9.txt_mlp.net.0.proj.lora_up.weight
transformer_blocks.9.txt_mlp.net.2.alpha
transformer_blocks.9.txt_mlp.net.2.lora_down.weight
transformer_blocks.9.txt_mlp.net.2.lora_up.weight
```

whereas, my custom trained and other LoRAs I've found on HF and civitai have keys like this

```
transformer.transformer_blocks.7.ff_context.net.2.lora_B.weight
transformer.transformer_blocks.7.norm1.linear.lora_A.weight
transformer.transformer_blocks.7.norm1.linear.lora_B.weight
transformer.transformer_blocks.7.norm1_context.linear.lora_A.weight
transformer.transformer_blocks.7.norm1_context.linear.lora_B.weight
```

I'm guessing that the prefix on the key is causing the issue for non-qwen-provided LoRAs?

**Comment by ghost** [comment]
**Use in Comfy this Versions**: 

_Python version: 3.12.9 
ComfyUI version: 0.3.49
ComfyUI frontend version: 1.24.4_

Use Comfy´s default **_LoraLoaderModelOnly_** Node and Run. (try Ksampler default first)

In my Case _smooth_ LoRA Train works. //3500 Steps/lr:3e-4/AE:1e-8/lrs:constant//
(depending on data set and resolution)

If lora is loaded but has no effect on the generation, the weights were not trained hard enough. Then try strength of 2-4 inner Pipeline. If the image is then irregular, the training has basically worked, but the loss did not fall deeply enough during training (no clear weight adjustments). Then repeat the training with more steps or a slightly more aggressive learning rate until the smooth loss has really fallen significantly and there are no more jumps or there is a risk of overfitting.

If Vram is less than 24 GiB, configure config.yaml with the following instructions:

_quantize: true
adam8bit: true
gradient_checkpointing: true_

fb-8 and bf-16 Version of Qwen-Image will accept the Trainingresult.

**Comment by machineminded** [comment]
@Shangooriginal  How did you resolve?
